src/Components/ContentDelivery/ContentDelivery.js

```diff
@@ -3,15 +3,27 @@ import style from './ContentDelivery.module.scss'
 import ReactPlayer from 'react-player'
 import { VideoContext } from '../../Context/VideoContext'
 import { ViewContext } from '../../Context/ViewContext'
+import { SettingsContext } from '../../Context/SettingsContext'
+import { SessionContext } from '../../Context/SessionContext'
 
 const ContentDelivery = () => {
   const [ done, setDone ] = useState(false)
   const [ videoLink ] = useContext(VideoContext)
   const [ view, setView ] = useContext(ViewContext)
+  const [ settings ] = useContext(SettingsContext)
+  const [ session, setSession ] = useContext(SessionContext)
```

Comment:
I find this session very interesting and clever!! What a great way to track information.